lib/listeners/statful.js

```diff
@@ -29,7 +19,7 @@ function dispatchMetricLine(listener) {
             });
 
             if (listener.config.stats) {
-                listener.statfulClient.counter('metrics_flushed', countMetricsInLines(lines));
+                listener.statfulClient.counter('metrics_flushed', linesList.length);
```

Comment:
I'm sorry, you're right. I was misreading `linesList` with `lines` and assuming you were getting the length of the original string.